contracts/MockContract.sol

```diff
@@ -325,11 +327,15 @@ contract MockContract is MockInterface {
 		calldataInvocations[keccak256(abi.encodePacked(resetCount, originalMsgData))] += 1;
 	}
 
-	fallback () payable external {
-		bytes4 methodId;
-		assembly {
-			methodId := calldataload(0)
-		}
+	receive() payable external {
```

Comment:
The changes here are actually unrelated to 0.7.0.

Reply:
this is a 0.6.x warning, right?

Reply:
Yes